src/WireMock.Net/Owin/AspNetCoreSelfHost.NETStandard13.cs

```diff
@@ -26,5 +28,22 @@ private static void SetHttpsAndUrls(KestrelServerOptions options, ICollection<(s
             }
         }
     }
+
+    internal static class IWebHostBuilderExtensions
+    {
+        internal static IWebHostBuilder ConfigureAppConfigurationUsingEnvironmentVariables(this IWebHostBuilder builder) => builder;

```

Comment:
Funny ;-)

Implementations in an interface , first time I see this in a real project ;-)